app/views/profiles/show.html.haml

```diff
@@ -1,4 +1,4 @@
-- content_for :meta_title, "#{@user.handle}'s profile on Exercism"
+- content_for :meta_title, "#{@user.handle}'s profile on Exercism".html_safe
```

Comment:
I don't think we can do this quite like this, as someone's handle might contain code injection.

I think instead we need:
```
"#{sanitize(@user.handle)}'s profile on Exercism".html_safe
```

But I can't remember what the actual `sanitize` method is, so you'll need to check and get the right one :)